frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -1,7 +1,7 @@
 import './ProjectListView.scss';
```

Comment:
I am currently adding more filters in for the reports to match the acceptance criteria - leaving PR as draft until then